docs/stdlib/doc/text.md

```diff
@@ -0,0 +1,207 @@
+## `capitalize`
+```ab
+pub fun capitalize(text: Text): Text 
+```
+
+Capitalize the first letter of the given `text`.
```

Comment:
just the first letter? like `quick fox` -> `Quick fox`? what about other languages than english?